ui/src/app/base/selectors/pod/pod.ts

```diff
@@ -137,7 +137,7 @@ const getHost = createSelector(
   [machine.all, controller.all, (_: RootState, pod: Pod) => pod],
   (machines: Machine[], controllers: Controller[], pod: Pod) => {
     if (!(pod && pod.host)) {
```

Comment:
You can use optional chaining here `!(pod?.host)`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
Adding that now